codegen/method.go

```diff
@@ -1204,9 +1210,6 @@ func (ms *MethodSpec) setWriteQueryParamStatements(
 		longQueryName := ms.getLongQueryName(field, thriftPrefix)
 		identifierName := CamelCase(longQueryName) + "Query"
 		_, isList := realType.(*compile.ListSpec)
-		if isList {
-			longQueryName = longQueryName + "[]"
```

Comment:
Let's update the doc https://github.com/uber/zanzibar/blob/master/docs/thrift.md#zanzibarhttpref 
> If the annotation is on a field of a list and that list is a method argument, the URL query name will be prefixed with the struct's field name plus "[]".

to reflect the change here.